mmtrack/models/task_modules/__init__.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 from .anchor import *  # noqa: F401,F403
+from .filter import *  # noqa: F401,F403
```

Comment:
We have `filter` both in `mmtrack/models/filter` and `mmtrack/models/task_modules/filter`.
Better to rename one of them.
Maybe `filter_modules` and `filter`?

Reply:
But we also have this convention: `mmtrack/models/motion` and `mmtrack/motion`